### PR TITLE
Add Speaker details

### PR DESCRIPTION
**Pull request by GoelJatin**

### CODE REVIEW

File: _data/speakers.yml [hunk 1]
```diff
@@ -184,4 +184,173 @@ jaidev_deshpande:
...
+      He builds data products to speed up adoption of ML in the enterprise.
+      Scribble Data, Dr. Pingali’s firm, offers a ML engineering platform and managed service.
+      Dr. Pingali has a BTech from IIT Mumbai and PhD from USC.
+    </p>
+
+pradhvan_bisht:
+  keynote: false
+  name: Pradhvan Bisht
+  social:
+    blog: https://blogs.dgplug.org/pradhvan/
```
Comment (GoelJatin) [inline _data/speakers.yml]: @ananyo2012 , do we have an svg for blogs that I could use here, worst case I'll go with Medium :P

Comment (bhansa) [reply _data/speakers.yml]: you can try this icon: https://fontawesome.com/icons/blog or the rss icon.

Comment (ananyo2012) [reply _data/speakers.yml]: Yeah please go ahead with RSS icon. If you don't see it working just use medium for now

**Review by ananyo2012 — APPROVED** [review]

File: _data/speakers.yml (hunk 1 above)
Comment (GoelJatin) [reply _data/speakers.yml]: used medium for now

**Comment by GoelJatin** [comment]
added speaker details shared over the email and their pics from their SM channels

**Comment by GoelJatin** [comment]
> added speaker details shared over the email and their pics from their SM channels

few speaker details are still missing

**Comment by GoelJatin** [comment]
@ananyo2012 @bhansa , can you please review and merge for speakers which we already have the details for.

**Comment by ananyo2012** [comment]
Looks good. Merging.

**Pull request merged**